src/System.IO.FileSystem/src/System/IO/FromApp.Interop.cs

```diff
@@ -0,0 +1,150 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.Win32.SafeHandles;
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+#pragma warning disable BCL0015
```

Comment:
Doh - yes. Then we could follow the pattern used by clrcompression, which I think makes it easier to see the all up view, which is a baseline with a note.
https://github.com/dotnet/corefx/blob/master/src/System.IO.Compression/src/PinvokeAnalyzerExceptionList.analyzerdata#L1

As for corelib in coreclr no such analyzer runs ATM (although the onecore one probably should). In Project N there is a scanner but I am not sure it runs nor uses the ID BCL0015 or baseline files.